docker/test/performance-comparison/compare.sh

```diff
@@ -359,7 +359,7 @@ create table unstable_run_traces engine File(TSVWithNamesAndTypes,
         'unstable-run-traces.$version.rep') as
     select
         count() value,
-        joinGet(addresses_join, 'name', arrayJoin(trace)) metric,
+        joinGet(addresses_join_$version, 'name', arrayJoin(trace)) metric,
```

Comment:
It is related, without this change performance tests script fails [here](https://github.com/ClickHouse/ClickHouse/pull/7512/files#diff-e5eafc5d8fe84bee7217583823e23336R638), because `clickhouse-local` leaves some data in working directory (however, without this change script just overwrites the data).